releng/release-notes-next/navy-linux.config

```diff
@@ -0,0 +1 @@
+Navy Linux 8 configuration fixed #1538
```

Comment:
Can you use this please?
`Navy Linux 8 configuration [fixed][issue#1538]`

Reply:
ok